packages/mask/src/plugins/RedPacket/SNSAdaptor/RedPacketERC20Form.tsx

```diff
@@ -199,10 +199,6 @@ export function RedPacketERC20Form(props: RedPacketFormProps) {
                             setRandom(e.target.value as number)
                         }}
                         MenuProps={{
-                            anchorOrigin: {
-                                vertical: 'top',
-                                horizontal: 'left',
-                            },
```

Comment:
I had the same problem when developed the tip feature.

I think no need to catch container via `usePortalShadowRoot` and do prop drilling.

## instead of

https://github.com/DimensionDev/Maskbook/blob/e46d955916051969c9d1479eb8f1b6e5385ec3aa/packages/mask/src/plugins/RedPacket/SNSAdaptor/RedPacketDialog.tsx#L247-L255

## suggest

https://github.com/DimensionDev/Maskbook/blob/e46d955916051969c9d1479eb8f1b6e5385ec3aa/packages/mask/src/plugins/NextID/components/Tip/TipForm.tsx#L128-L146